code/ShutinaSvetlana/batcher.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import math
+
+
+class Sorting:
+    # class realizes the sorting for array of numbers
+    arr = []
+
+    def __init__(self, arr):
+        self.arr = arr
```

Comment:
What will happen if we will not give the correct data? Please add tests for this.